### PR TITLE
New psets for Phase2PixelNtuple 

### PR DESCRIPTION
**Pull request by emiglior**
This is an update of PR #22204. Two psets are provided:
1/ phase2_reco_pixelntuple_cfg.py: ntuples from DIGI to perform RECO level studies, e.g. CPE errors calibration)
2/ phase2_digi_reco_pixelntuple_cfg.py ntuples from SIM to produce "stimuli" for RD53 studies (InnerTracker PROC deisgn)

### CODE REVIEW

File: SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+This is a short description of the content of the ntuple produced by Phase2PixelNtuple.cc
```
Comment (kpedro88) [inline SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md]: please actually use markdown in this file so it is readable

**Comment by kpedro88** [comment]
please test

**Comment by ianna** [comment]
+1

File: SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md (hunk 1 above)
Comment (boudoul) [reply SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md]: hi @emiglior , I guess this comment should be adressed , thanks.

Comment (emiglior) [reply SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md]: @kpedro88 sorry for the slow reaction... any suggestion, reference, cmssw example for md style is welcome

**Comment by kpedro88** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**